### PR TITLE
PageLayout and Togglebox improvements

### PR DESCRIPTION
**Pull request by fritzSF**

### CODE REVIEW

File: app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx [hunk 1]
```diff
@@ -38,6 +42,7 @@ export class CustomHeaderToggleBox extends React.PureComponent<IPropsCustomHeade
                     <button
                         className='new-collapse-box__divider'
                         onClick={this.toggle}
+                        aria-controls={`togglebox-${this.htmlId}`}
```
Comment (tomaskikutis) [inline app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx]: pass an argument to `nextId` so you don't need to concatenate it in all usages.

File: app-typescript/components/ToggleBox/SimpleToggleBox.tsx [hunk 2]
```diff
@@ -75,23 +79,9 @@ export class SimpleToggleBox extends React.PureComponent<IPropsSimple, IState> {
                     {badge ? badge : null}
                 </a>
                 <div className="toggle-box__content-wraper">
-                    {isOpen && !hideUsingCSS && (
-                        <div className="toggle-box__content" aria-describedby={`togglebox-${this.htmlId}`}>
-                            {children}
-                        </div>
-                    )}
-
-                    {hideUsingCSS && (
```
Comment (tomaskikutis) [inline app-typescript/components/ToggleBox/SimpleToggleBox.tsx:84]: if you're removing the implementation also drop other usages including in props and docs

Comment (fritzSF) [reply app-typescript/components/ToggleBox/SimpleToggleBox.tsx:84]: Done

**Comment by tomaskikutis** [comment]
can you address the remaining(unresolved) comment as well?

File: app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx (hunk 1 above)
Comment (fritzSF) [reply app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx]: Done.

File: app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ interface IState {
 }
 
 export class CustomHeaderToggleBox extends React.PureComponent<IPropsCustomHeader, IState> {
-    htmlId = nextId();
+    htmlId = "togglebox-" + nextId();
```
Comment (tomaskikutis) [inline app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx]: "togglebox" could also be passed as an argument to `nextId`. It doesn't change much, just a bit cleaner

**Review by tomaskikutis — APPROVED** [review]

File: app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx (hunk 3 above)
Comment (fritzSF) [reply app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx]: I get now what you mean. It is cleaner, I will change it

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**